next.config.js

```diff
@@ -57,10 +57,24 @@ module.exports = {
         destination: '/legacy?path=/campaign/action%3A:eventId&orgId=:orgId',
         permanent: false,
       },
-      // all paths with /o redirected to Gen2
       {
-        source: '/o/:path*',
-        destination: `http://${process.env.ZETKIN_API_DOMAIN}/o/:path*`,
+        source: '/o/:orgId',
+        destination: `http://${process.env.ZETKIN_API_DOMAIN}/o/:orgId`,
+        permanent: false,
+      },
+      {
+        source: '/o/:orgId/events/:eventId',
+        destination: `http://${process.env.ZETKIN_API_DOMAIN}/o/:orgId`,
+        permanent: false,
+      },
+      {
+        source: '/o/:orgId/map',
+        destination: `http://${process.env.ZETKIN_API_DOMAIN}/o/:orgId/map`,
+        permanent: false,
+      },
```

Comment:
I just added one of these each for all the placeholder pages from the previous Code Camp. Wasn't sure which ones were needed to avoid breaking existing stuff. Have removed the `/o/:orgId/map` one now as it's clear that one's not needed. LMK if there's others worth removing too.